src/components/menu/menu.svelte

```diff
@@ -223,6 +223,8 @@
   }
 
   .leo-menu .leo-menu-popup {
+    --min-width: var(--leo-menu-control-min-width, var(--leo-menu-target-width));
```

Comment:
brave rewards codebase uses the previous name so we will need to change it to this when updating nala

Reply:
Good to know!